lib/Models/ItemSearchProviders/TextIndex.ts

```diff
@@ -49,14 +49,18 @@ export default class TextIndex implements IndexBase<TextSearchQuery> {
   /**
    * Search the text index for the given value.
    *
-   * @param value The value to be searched.
+   * @param value         The value to be searched.
+   * @param queryOptions  MiniSearch.SearchOptions
    * @param return The IDs of objects matching the search text.
    */
-  async search(value: TextSearchQuery): Promise<Set<number>> {
+  async search(
+    value: TextSearchQuery,
+    queryOptions: any
```

Comment:
For better typing, use the more specific MiniSearch.SearchOptions type here and cast the caller's input param to `MiniSearch.SearchOptions`